R/cancensus.R

```diff
@@ -325,7 +325,21 @@ cancensus.list_vectors <- function(dataset) {
   result %>%
     dplyr::mutate(type = factor(type),
                   units = factor(units)) %>%
-    dplyr::select(vector, type, label, units, parent_vector = parent)
+    dplyr::select(vector, type, label, units, parent_vector = parent) %>%
+    dplyr::mutate(
+      units = recode(.$units, 
+                         `1` = "number",
+                         `2` = "percentage ratio (0.0-1.0)",
+                         `3` = "currency",
+                         `4` = "ratio",
+                         `5` = "percentage (0-100)")) %>%
+    dplyr::mutate(
+      add = recode(.$add,
+                         `1` = "Additive",
+                         `2` = "Averages",
```

Comment:
Ok, that's good to know. In that case, it shouldn't be stored as a factor. Instead of using `recode`, can use `case_when` here and make the tests a little bit more flexible, so it will pick up the ".xxxx" and call it an average, regardless. Unless you want to be something more specific?